annotator/templates/video.html

```diff
@@ -51,7 +53,7 @@
                 {% if assignment_id != None %}
                 <form method="post" id="turk-form" action="https://{{ MTURK_SANDBOX|yesno:'workersandbox,www'}}.mturk.com/mturk/externalSubmit" style="margin-left: 300px">
```

Comment:
There are 3 modes we can operate in:
- standalone, without mturk. This is what we normally test on. `iframe_mode` is `False`, `assignment_id` is `None`
- mturk preview. `iframe_mode` is `True`, `assignment_id` is `None`, `preview` is `True`. Should be same as mturk mode but no submit button.
- mturk mode. `iframe_mode` is `True`, `assignment_id` is an id string, `preview` is `False`.